### ISSUE TITLE
is port forwarding really required?

### ISSUE THREAD

**Issue by marqueeeeeee** [williampiat3/ImprovingPSRemotePlay#8]
I was following the instruction on implementing bridge but it still does require port forwarding. My problem is that my ISP implements CGNAT so I cannot do any port forwarding and I thought VPN was the solution for this. Any thoughts?

**Comment by williampiat3** [comment]
Hi Marqueeeee, thanks for your message it seems like you are having a hard time with your ISP. 
Does your router can connect to a VPN? Some routers can directly transfer the whole traffic through a VPN and thus if you host your VPN bridge server on another network (or a cloud platform) you could connect both your ps5 and your computer.
This might impact performance, though.
If it is not feasible, I would advise you to contact you ISP: one should be able to host a small website on a local server it should be feasible to disable CGNAT for a limited amount of users but yes it might be cumbersome knowing how unhelpfull they can be.

**Comment by williampiat3** [comment]
Or you could use Tailscale (we have a guide for the install) , it is using an external server to forward every packet this should work but you will have to use a TUN VPN and thus Chiaki

**Comment by Pimss** [comment]
You will find all info on how to set up tailscale [here](https://github.com/williampiat3/ImprovingPSRemotePlay/blob/main/tailscale_install.md)
Similarly, you can set up Chiaki following this [part](https://github.com/williampiat3/ImprovingPSRemotePlay#install-chiaki-) of the guide.
Some ISP now lock port forwarding by default but you can, in specific cases, ask them to restaure this feature.

**Comment by marqueeeeeee** [comment]
Yup, i have successfully implemented tailscale and subnet and work's great so far. Thank you!
